moveit_commander/src/moveit_commander/move_group.py

```diff
@@ -579,8 +579,9 @@ def set_max_acceleration_scaling_factor(self, value):
                 "Expected value in the range from 0 to 1 for scaling factor"
             )
 
-    def go(self, joints=None, wait=True):
-        """ Set the target of the group and then move the group to the specified target """
+    def go(self, joints=None, wait=True, planning_pipeline=None, planner_id=None):
+        """Set the target of the group and then move the group to the specified target.
+        planning_pipeline and planner_id specify the planner that is used for this motion."""
```

Comment:
Could you also explain what the `wait` variable means?

Reply:
Sure